File: wiseclientlibrary/src/main/java/com/wiseclient/script/DisplayScript.java
package com.wiseclient.script;

import javax.swing.JTextArea;

/**
 * Created by kuanghaochuan on 2017/7/17.
 */

public class DisplayScript implements ScriptInterface {
    private JTextArea mJTextArea;

    DisplayScript(JTextArea jTextArea) {
        this.mJTextArea = jTextArea;
    }

    @Override
    public void handleScript(StringBuilder stringBuilder) {
        this.mJTextArea.append(stringBuilder.toString());
    }
}

File: wiseclientlibrary/src/main/java/com/client/script/DisplayScript.kt
package com.client.script

import javax.swing.JTextArea

/**
 * Created by kuanghaochuan on 2017/7/17.
 */

class DisplayScript internal constructor(private val mJTextArea: JTextArea) : ScriptInterface {

    override fun handleScript(stringBuilder: StringBuilder) {
        this.mJTextArea.append(stringBuilder.toString())
    }
}
